### PR TITLE
add validation around the resurrection time being sooner than the cur…

### LINKED ISSUE #137: investigate failing createSarcophagus call with archaeologist 4

### PR DESCRIPTION
**Pull request by gh-23378**
…rent date + max interval

### CODE REVIEW

File: src/lib/utils/helpers.ts [hunk 1]
```diff
@@ -94,9 +94,9 @@ export function humanizeUnixTimestamp(unixTimestamp: number): string {
   return new Date(unixTimestamp).toLocaleDateString('en-US');
 }
 
-export function humanizeDuration(duration: number) {
-  if (duration === 0) return '0 seconds';
-  return moment.duration(duration / 1000).humanize({ d: 7, w: 4 });
+export function humanizeDuration(durationMs: number) {
+  if (durationMs === 0) return '0 seconds';
+  return moment.duration(durationMs).humanize({ d: 7, w: 4 });
```
Comment (sethhrbek) [inline src/lib/utils/helpers.ts:99]: @katien  --- Looks like this function is being used by an embalmer step that was removed from UI -- `SetDiggingFees` -- maybe both this function and that step code (which still remains) should be removed?

File: src/features/embalm/stepContent/hooks/useRecoverPublicKey.ts [hunk 2]
```diff
@@ -99,14 +99,14 @@ export function useRecoverPublicKey() {
 
           return;
         }
-
+        console.log(`retrieving public key for address ${address}`);
```
Comment (knoll3) [inline src/features/embalm/stepContent/hooks/useRecoverPublicKey.ts]: If we will be using console logs in development lets just not forget to override them for production.

Comment (sethhrbek) [reply src/features/embalm/stepContent/hooks/useRecoverPublicKey.ts]: there is a basic `log` function we could start using, and purposefully gate the logging in that function to environments we define (like development). Will probably be useful to have these logs on testnet webapp, but can remove them for mainnet.

Comment (knoll3) [reply src/features/embalm/stepContent/hooks/useRecoverPublicKey.ts]: We can continue to use `console.log`, just override the `console.log` function based on the environment: 
```
if (process.env.NODE_ENV === 'production') {
  console.log = () => {}
}
```

**Review by knoll3 — APPROVED** [review]

File: src/features/embalm/stepContent/hooks/useRecoverPublicKey.ts (hunk 2 above)
Comment (gh-23378) [reply src/features/embalm/stepContent/hooks/useRecoverPublicKey.ts]: Good call! I swapped this file over to Seth's logger and updated it to stay silent on prod. We'll have a fair amount of `console.log`s to update at some point but I didn't bother with them here.

File: src/lib/utils/helpers.ts (hunk 1 above)
Comment (gh-23378) [reply src/lib/utils/helpers.ts:99]: Creating another task to remove dead code because we've got a significant amount in this repo

**Pull request merged**